server/views/applications/pages/basic-information/pause-application.njk

```diff
@@ -0,0 +1,31 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+
+{% extends "../../../partials/layout.njk" %}
+
+{% set pageTitle = applicationName + " - " + pageHeading  %}
+{% set mainClasses = "app-container govuk-body" %}
+
+{% block content %}
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            {{ govukBackLink({
+		        text: "Back",
+		        href:  paths.applications.pages.show({
+                    id: applicationId,
+                    task: 'basic-information',
+                    page: 'board-taken-place'
+                })
+	        }) }}
+            <h1 class="govuk-heading-l">{{page.title}}</h1>
+            <p class="govuk-body">From the information you’ve provided the person requires a complex case board needs to take place before you can apply for an Approved Premises (AP) placement.</p>
```

Comment:
Might be good for us to somehow add a `paused` status to an application for things like this. Fine to ship as-is, but something for us to ponder.

Reply:
yeah, good point